build/rust/BUILD.gn

```diff
@@ -21,6 +21,10 @@ cargo_build("rust_lib") {
     "//brave/components/speedreader/rust/lib",
     "//brave/vendor/adblock_rust_ffi:rust_lib",
   ]
+
+  if (is_win) {
+    deps += [ "//brave/components/rpill" ]
```

Comment:
this is not correct, you need a dep for lib itself here. Please follow the patterns from speedreader

Reply:
Not using rust anymore.